### PR TITLE
Issue #1418: Selecting Unique User records

### PR DESCRIPTION
**Pull request by natseg**
Hi,

I was able to reproduce issue #1418.

From a UX perspective, not being able to pick a unique record can be problematic.
This PR impacts other places where the value for the user record was first name + last name prior to commit.
My opinion is in favour of displaying unique records and human readable records for user selection.

screen shot 2017-09-17 at 11 20 18

versus

screen shot 2017-09-17 at 23 57 57

I have also corrected really minor typos.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -277,13 +277,14 @@ GEM
     omniauth-twitter (1.4.0)
       omniauth-oauth (~> 1.1)
       rack
+    parallel (1.12.0)
     parser (2.4.0.0)
       ast (~> 2.2)
-    pg (0.20.0)
+    pg (0.21.0)
```
Comment (thorsteneckel) [inline Gemfile.lock]: Please don't perform `bundle update` since some dependency versions (e.g. `pg`) are currently broken 😞

Comment (natseg) [reply Gemfile.lock]: About the changes, I went with the user's suggestion, but yes, showing the email address on hover would make sense.
I will see what I can do later today. 👍

Comment (natseg) [reply Gemfile.lock]: About the changes, I went with the user's suggestion, but yes, showing the email address on hover would make sense.
I will see what I can do later today. 👍

**Comment by natseg** [comment]
About the changes, I went with the user's suggestion, but yes, showing the email address on hover would make sense.
I will see what I can do later today. 👍

**Comment by natseg** [comment]
Hi!
I hope showing the email as a tooltip using attribute title is acceptable.

Thanks

**Comment by thorsteneckel** [comment]
Nice! Could you please reset your changes to the `Gemfile.lock` as I stated in my comment so I can test the pull request? Thanks!

**Comment by thorsteneckel** [comment]
Hi @natseg - Sorry for the delay. Could you please update your PR branch with the latest state of develop? There are some changes that your branch will undo if merged that are out of the scope of this PR.

**Comment by thorsteneckel** [comment]
Hi @natseg - Thanks for updating the commit. I did some rework to your changes. You can review them above. Thanks for your contribution 🚀

**Pull request closed without merging**